### ISSUE TITLE
Rename classes/methods

### ISSUE THREAD

**Issue by sfshaza2** [flutter/flutter#23705]
We have been asked to remove instances of "Blacklist" and "Whitelist" from the docs. I searched the API docs and found two classes:

https://docs.flutter.io/flutter/services/WhitelistingTextInputFormatter-class.html
https://docs.flutter.io/flutter/services/BlacklistingTextInputFormatter-class.html

And methods belonging to these classes. Please reconsider renaming.

Suggested replacements:
denylist/allowlist or blocklist/allowlist

Basically, use what makes sense for the domain.

**Comment by miquelbeltran** [comment]
Seems that the fix is coming here: https://github.com/flutter/flutter/pull/59120 🎉

**Comment by iapicca** [comment]
Hi @sfshaza2 
despite the [PR](https://github.com/flutter/flutter/pull/59120) being merge I see in the latest [master](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/text_formatter.dart) the classes containing the terms indicated

<details><summary>logs</summary>

```console
[✓] Flutter (Channel master, 1.21.0-6.0.pre.17, on Mac OS X 10.15.5 19F101, locale en-GB)
    • Flutter version 1.21.0-6.0.pre.17 at /Users/nevercode/development/flutter_master
    • Framework revision e64fe11542 (12 hours ago), 2020-07-23 18:11:17 -0700
    • Engine revision 626244a72c
    • Dart version 2.9.0 (build 2.9.0-21.0.dev a3815b6590)


[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/nevercode/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.5)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.5, Build version 11E608c
    • CocoaPods version 1.9.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 46.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.2

[✓] Connected device (3 available)
    • macOS (desktop)  • macos      • darwin-x64     • Mac OS X 10.15.5 19F101
    • Web Server (web) • web-server • web-javascript • Flutter Tools
    • Chrome (web)     • chrome     • web-javascript • Google Chrome 84.0.4147.89

• No issues found!
```

</details>
Thank you
